ando/rulesStructured.py

```diff
@@ -41,19 +41,21 @@
 currentdepth_rules['authorized_folders'] = ['^$']
 currentdepth_rules['mandatory_folders'] = []
 currentdepth_rules['authorized_data_files'] = [
-    ['^sub-([a-zA-Z0-9]+)_ses-([a-zA-Z0-9]+)([\\w\\-]*)_ephys', ANY_DATA_EXT]]
+    ['sub-([a-zA-Z0-9]+)_ses-([a-zA-Z0-9]+)([\\w\\-]*)(_task-([a-zA-Z0-9]+))?'
```

Comment:
You removed the '^' at the beginning of the regexp patterns, that are anchoring the patterns to the beginning of the strings to match. I don't think this was intended...